src/packaging/bin/cassandra-reaper

```diff
@@ -22,7 +23,7 @@ fi
 
 if [ -z "$CLASS_PATH" ]; then
   echo "Looking for reaper in /usr/share/cassandra-reaper/"
-  CLASS_PATH="/usr/share/cassandra-reaper/*:/usr/local/share/cassandra-reaper/*"
+  CLASS_PATH="$(find /usr/local/share -path 'cassandra-reaper/cassandra-reaper-*[0-9rT].jar'):$(find /usr/share -path 'cassandra-reaper/cassandra-reaper-*[0-9rT].jar')"
```

Comment:
If operators were intentionally putting other jar files into the `.../share/cassandra-reaper/` folder, eg log or metric spi implementations, then this will break that as it now only loads the cassandra-reaper shaded jar file.

The correct approach for operators in such a situation is to manually declare `$CLASSPATH`.